src/accordion/accordion-group.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @ngdoc directive
+ * @name bdAccordionGroup
+ * @module buildium.angular-elements.accordion
+ * @restrict A
+ *
+ * @description
+ * 
+ * Encapsulates a section of content that can be toggled by {@link buildium.angular-elements.accordion.directive:bdAccordionToggle bdAccordionToggle}
+ *
+ * @param {String} openClass - class to add when accordion group is open
+ * 
+ * @example
+ *
+    <example name="bd-accordion-group" module="buildium.angular-elements.accordion">
```

Comment:
did anything change in any of the accordion files from how they were in buildiumcode? Or basically just porting them over and adding the ng-docs?

Reply:
I've ported them over as-is, made them pass linting and added ng-docs

I've also changed the names of the CSS classes, which used to be prefixed with "_bd-_"